binderhub/app.py

```diff
@@ -332,6 +334,7 @@ def _add_slash(self, proposal):
             'gist': GistRepoProvider,
             'git': GitRepoProvider,
             'gl': GitLabRepoProvider,
+            'zn': ZenodoProvider,
```

Comment:
I regret not spelling these out earlier :D can we spell 'zenodo' out here?

Reply:
Has anyone complained about them being short names? I am undecided if I like or dislike the short names.

Reply:
zn to me is the language code for Chinese rather than zenodo :D I think we should be as clear as possible...